src/olympia/blocklist/templates/admin/blocklist/includes/logs.html

```diff
@@ -21,7 +21,7 @@
         </ul>
       </li>
     {% endfor %}
-    {% if blocklistsubmission %}
+    {% if blocklistsubmission|default_if_none:False %}
```

Comment:
driveby to attempt to stop complaints about undefined variables in django templates